src/UserInterfaceServiceProvider.php

```diff
@@ -21,12 +21,12 @@ class UserInterfaceServiceProvider extends ServiceProvider
     public function register()
     {
         Flash::levels([
-            'info'    => 'alert-info',
-            'success' => 'alert-success',
-            'warning' => 'alert-warning',
-            'danger'  => 'alert-danger',
-            'error'   => 'alert-error',
-            'hint'    => 'alert-hint',
+            'info'    => 'info',
+            'success' => 'checkmark-smooth',
+            'warning' => 'exclamation-mark',
+            'danger'  => 'cross',
+            'error'   => 'cross',
+            'hint'    => 'question-mark',
```

Comment:
doesn't this have to do with the classes applied to the flash messages?

Reply:
It does, had to check the Spatie doc, thought it'd be about the icons, my bad